hw/top_earlgrey/ip/sensor_ctrl/doc/_index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Sensor Control Technical Specification"
+---
+
+# Overview
+
+This document specifies the functionality of the sensor control module.
+The sensor control module is a comportable front-end to the [analog sensor top]({{< relref "hw/top_earlgrey/ip/ast/doc" >}}).
+
+It provides basic alert functionality, pad debug hook ups, and a small amount of open source visible status readback.
+Long term, this is a module that can be absorbed directly into the `analog sensor top`.
+
+## Features
+
+- Alert hand-shake with `analog sensor top`
+- Alert forwarding to `alert handler`
+- Status readback for `analog sensor top`
+- Pad debug hook up for `analog sensor top`
+
+# Theory of Operations
+
+## Block Diagram
+
+The diagram below shows how `sensor control` helps `analog sensor top` integration into the overall design.
+
+## Alert Handshake
+
+The `analog sensor top` sends alert requests in independent, differential form to the `sensor control`.
+Each alert request is consisted of a pair of signals, one active high and one active low.
+However, the active polarity of each signal is independent, this reflects the imprecise timing sensor that drives the alert.
```

Comment:
nit: two unrelated sentences seem to be joined with a comma

Reply:
updated